src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1256,7 +1256,7 @@ void window_guest_stats_update(rct_window* w)
  *
  *  rct2: 0x0066ECC1
  *
- *  ebp: colour, contains flag BAR_BLINK for blinking
+ *  colour: contains flag BAR_BLINK for blinking
```

Comment:
can you look into seeing if this can be refactored into a separate variable. There is no need to have it combined into colour.

Reply:
Sure, will update.